UKHO.ExchangeSetService.API/UKHO.ExchangeSetService.API.FunctionalTests/FunctionalTests/ExchangeSetForLargeFile.cs

```diff
@@ -38,6 +35,8 @@ public async Task SetupAsync()
             ScsJwtToken = await authTokenProvider.GetScsToken();
             DataHelper = new DataHelper();
             ApiEssResponse = await ExchangeSetApiClient.GetProductIdentifiersDataAsync(DataHelper.GetProductIdentifiersForLargeMedia(), accessToken: EssJwtToken);
+            //This will allow to donwload the files
+            await Task.Delay(5000);
```

Comment:
remove and refactor as in the previous way of working. we agreed that there will be no fixed delays and that it needs to be dynamic. there is of course nothing wrong with having an overarching test timeout.

Reply:
Hi,

We have removed this delay and added it to the proper location in download method.

Regards,
Mayuresh Gawde